main.js

```diff
@@ -27,21 +27,12 @@ $(document).ready(function () {
             break;
     }
 
-    // check release time to inform people in case they are running old release
-    if (CONFIGURATOR.releaseDate > (new Date().getTime() - (86400000 * 91))) { // 1 day = 86400000 miliseconds, * 91 = 3 month window
-        console.log('Application version is valid for another: ' + Math.round((CONFIGURATOR.releaseDate - (new Date().getTime() - (86400000 * 60))) / 86400000) + ' days');
-    } else {
-        console.log('Application version expired');
-        GUI.log('You are using an old version of ' + chrome.runtime.getManifest().name + '. There may be a more recent version with improvements and fixes.');
-    }
-
```

Comment:
release date is not maintained correctly ... you should maintain or delete it

last entry:
`'releaseDate': 1463055533515, // new Date().getTime() - Thu May 12 2016 14:19:06 GMT+0200`